src/executors/mod.rs

```diff
@@ -19,12 +21,30 @@ fn get_function_output<'a>(
     let handler = function.handler.as_ref(py);
     let request_hashmap = request.to_hashmap(py).unwrap();
 
-    // this makes the request object accessible across every route
-    match function.number_of_params {
+    if function.validate_params {
+        let body = request.body.clone().to_vec();
+        let json_str = str::from_utf8(&body).unwrap();
+
+        // Use Python to convert back to PyDict after deserializing
+        let json = py.import("json").unwrap();
+
+        // Extract the route's dependencies from the FunctionInfo
+        let json_dict = json.call_method1("loads", (json_str,)).unwrap();
+
+        // Get kwargs from the PyDict
+        let kwargs: &PyDict = json_dict.try_into().unwrap();
+
+        let test = handler.call((), Some(kwargs)).unwrap();
```

Comment:
Let's give this variable a better name than `test`? Something like response/output?